e2e/suite_test.go

```diff
@@ -40,54 +33,74 @@ func Test_All(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	scenarios := scenario.AllScenarios()
 	if err := createMissingClusters(ctx, r, scenarios, &clusterConfigs); err != nil {
```

Comment:
I guess if we do it this way then the suite will attempt to create testing clusters for scenarios that we might not actually be running

I'm not necessarily opposed to this since in theory that would speed up subsequent runs either from the same user if they decide to later run the scenarios they originally skipped, or if other users run those scenarios as well

Reply:
I think I can do something similar to the way I load last version for SIG. Create a cluster in a lazy way and only once.

But I'd rather fix it in a separate PR. This one already grew beyond my intent.